### PR TITLE
Update demo.py

### PR DESCRIPTION
**Pull request by gouravcy**
If the probability of any label is less than 0.5 than we can show both the best and the second best prediction using above code.

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -62,32 +62,73 @@ def demo(opt):
...
                 # select max probabilty (greedy decoding) then decode index to character
                 _, preds_index = preds.max(2)
                 preds_str = converter.decode(preds_index, length_for_pred)
+                print(preds_str)
+
+            preds_prob = F.softmax(preds, dim=2)
+            preds_max_prob, preds_index_prob = preds_prob.max(dim=2)
+            print(preds_max_prob)
+            print(preds_max_prob.type)
+            index=np.zeros((11,26))
```
Comment (seunghwanly) [inline demo.py:74]: Hi, 🙋‍♂️ just looking at line 74
is `import numpy as np` needed at the front?
